packages/aft/lib/src/commands/deps_command.dart

```diff
@@ -235,6 +275,8 @@ class _DepsUpdateCommand extends _DepsSubcommand {
       exitCode = 1;
     }
 
-    await _run(_DepsAction.apply);
+    if (hasUpdates && failedUpdates.isEmpty) {
+      await _run(_DepsAction.apply);
```

Comment:
Do we need to warn here if failedUpdates was not empty?

Reply:
Yep, happens on the line above